healthCare-Frontend/src/components/assessment-form/stepFive/StepFive.js

```diff
@@ -133,6 +133,49 @@ const StepFive = () => {
     navigate("/step-four");
   };
 
+  const educationalDiscussionData = [
```

Comment:
instead of creating this constant array in direct step component file, create a new `labels` folder inside `utils` folder and create separate file for each of the step component like `stepOneLabels.js`and export the constants array from there